apps/modernization-ui/src/apps/patient/add/extended/useAddExtendedPatient.ts

```diff
@@ -66,6 +66,8 @@ const useAddExtendedPatient = (): AddExtendedPatientInteraction => {
                 subFormDirtyState.race
             ) {
                 dispatch({ type: 'invalidate', validationErrors: { dirtySections: subFormDirtyState } });
+                const form = document.getElementById('extended-form-errors');
+                form?.scrollIntoView({ behavior: 'smooth', block: 'start' });
```

Comment:
question:  Should the hook have knowledge of the structure of one of it's callers?